6_drivers/src/nldas_feather_utils.R

```diff
@@ -8,8 +8,9 @@ create_cell_task_plan <- function(cells, time_range, cube_files, ind_dir, feathe
     command = function(target_name, task_name, ...) {
       this_var <- parse_feather_filename(task_name, 'var') # better filtering needed?
       these_files <- cube_files[grepl(this_var, cube_files)] %>%
+        basename() %>% paste0('.ind') %>% file.path(cube_ind_dir, .) %>%
         paste("\n       \'", ., collapse = "\',", sep = "")
-      sprintf('cubes_to_cell_file(target_name, %s\')', these_files)
+      sprintf('cubes_to_cell_file(target_name, nc_dir = I(\'%s\'), %s\')', cube_nc_dir, these_files)
```

Comment:
It's sorta awkward that this line has to add an orphaned single quote just before the closing parenthesis to cubes_to_cell_file. It does work, but if you wanted an alternative, you could change line 12 to
```
paste0("\n       '", ., "'", collapse = ",")
```
and then leave out that orphaned quote in this line.